code/modules/jobs/job_mail.dm

```diff
@@ -346,6 +351,9 @@
 		/obj/item/toy/figure/hos = 5,
 		/obj/item/reagent_containers/food/drinks/drinkingglass/quintuple_sec = 4,
 		/obj/item/megaphone/command/rudiger = 4,
+		/obj/item/clothing/shoes/jackboots/fast = 2,
+		// You want it don't you?
+		/obj/item/melee/classic_baton/police/telescopic = 1,
```

Comment:
This powerful item is supposed to be restricted to heads of staff spawns

Reply:
You already get one, just not telescopic, he also is a head and doesn't get one. Let him dream?